oneview_redfish_toolkit/api/blade_manager.py

```diff
@@ -39,9 +39,9 @@ def __init__(self, server_hardware):
 
         super().__init__(server_hardware, server_hardware['mpFirmwareVersion'])
 
-        self.redfish["FirmwareVersion"] = server_hardware['mpFirmwareVersion']
```

Comment:
Why was this removed?

Reply:
it is passed into the super class and stored.  This is a duplicate and should have been removed.